src/Rings/mpoly-ideals.jl

```diff
@@ -46,15 +47,15 @@ function ideal(R::MPolyRing, g::Vector)
   return ideal(f)
 end
 
-function ideal(Qxy::MPolyRing{T}, x::MPolyElem{T}) where T <: RingElem 
+function ideal(Qxy::MPolyRing{T}, x::MPolyElem{T}) where T <: RingElem
```

Comment:
Can the white space changes please either be removed or be put into a commit of their own?